pkg/ephemeral/player.go

```diff
@@ -223,5 +223,6 @@ func (c *Callbacker) sendEvent(name, topic string, e interface{}) {
 		},
 	}
 	c.pb.PublishWithBody(name, topic, event, c.playerParams.GameID)
-	c.logger.Debugf("Sending event %v to topic %s\n", event.Name, topic)
+	c.logger.Debugw("Sending event", "event", event, "topic", topic)
+	c.logger.Debugf("Sending event.name %v to topic %s\n", event.Name, topic)
```

Comment:
Copy-paste error?

Reply:
I'd say we keep the `Debugw` call since it contains more information than just the event name and topic